uvahelpr_exp/uvahelpr_exp/views.py

```diff
@@ -29,9 +29,8 @@ def job_summary(request, id):
     lastname = user_info['last_name']
     name = firstname + ' ' + lastname
 
-    newjob = {x: job[x] for x in ['id', 'title', 'description', 'time_required', 'event_time', 'skills_required', 'compensation', 'location']}
-    newjob['requester'] = name
+    result = {x: job[x] for x in ['id', 'title', 'description', 'time_required', 'event_time', 'skills_required', 'compensation', 'location']}
+    result['requester'] = name
 
-    result = json.dumps(newjob)
-    return HttpResponse(result)
+    return JsonResponse(result)
```

Comment:
Referencing the previous comment, we should also keep a consistent format and do 'result' => result and another status key-val pair.